.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run type-check && npm test
```

Comment:
Currently there is lint check in pre push. Can dropping it cause any problems?

Reply:
The linter will run on the files you have staged when you commit, so I don't see the need to run before pushing, only if you skip the hook when commiting.